controllers/openstackcluster_controller.go

```diff
@@ -590,6 +590,60 @@ func bastionHashHasChanged(computeHash string, clusterAnnotations map[string]str
 	return latestHash != computeHash
 }
 
+func reconcileLoadBalancerNetwork(openStackCluster *infrav1.OpenStackCluster, networkingService *networking.Service) error {
```

Comment:
Can we pull out:

```golang
lbSpec := openStackCluster.Spec.APIServerLoadBalancer
```

up at the top? It would make some of these lines much shorter.